mode-lambda/backend/gl/ngl.fragment.glsl

```diff
@@ -21,12 +21,13 @@ void main(void)
   vec2 texSize = textureSize(SpriteAtlasTex, 0);
   SpriteColor =
     // This is what it should be defined as
-    //  texture(SpriteAtlasTex,
-    //          TexCoord / float(texSize.x))
+    texture(SpriteAtlasTex,
+            vec2(TexCoord.x / texSize.x,
+                 TexCoord.y / texSize.y))
     // But it doesn't work on some ES devices I have, so we do this instead:
-    texelFetch(SpriteAtlasTex,
-               ivec2( trunc(TexCoord.x), trunc(TexCoord.y) ),
-               0)
+    //texelFetch(SpriteAtlasTex,
+    //           ivec2( trunc(TexCoord.x), trunc(TexCoord.y) ),
+    //           0)
```

Comment:
Could you put these changes with an `if true/false` and then I can go in and make it compile-time?